scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -142,14 +142,43 @@ elif [ "$os" == "ubuntu20.04" ] ; then
 elif [ "$os" == "rhel8" ] ; then
 
     pkgs_opencv_req=(gtk3)
-    pkgs_python=(python3 python3-devel python3-setuptools python3-pip)
-    pkgs_dev=(gcc gcc-c++ make glibc libstdc++ libgcc cmake pkg-config gflags-devel.i686 zlib-devel.i686 curl sudo)
+    pkgs_python=(python38 python38-devel python38-setuptools python38-pip)
+    pkgs_dev=(gcc gcc-c++ make glibc libstdc++ libgcc cmake pkg-config zlib-devel.i686 curl sudo)
     pkgs_myriad=()
     pkgs_opencv_opt=(
-        gstreamer1
-        gstreamer1-plugins-bad-free
-        gstreamer1-plugins-good
-        gstreamer1-plugins-ugly-free
+        https://vault.centos.org/centos/8/AppStream/x86_64/os/Packages/libcdio-2.0.0-3.el8.x86_64.rpm
```

Comment:
in near future we want to eliminate this script and provide rpm packages where all dependencies will be provided in the appropriate sections of the package. In this case we need to use default dependencies on operation system.
How will this work then?

Reply:
BTW, since we have decided to go with community OpenCV version in the 2022.1.1 release, do we need OpenCV dependencies at all?

Reply:
> in near future we want to eliminate this script and provide rpm packages where all dependencies will be provided in the appropriate sections of the package. In this case we need to use default dependencies on operation system. How will this work then?

This change is updating the method of installation to be independent from RH subscription. Not all packages are available in the distribution repository some potentially some would be dropped in the future like gstreamer. Note that currently this script is added the dependencies to in OV docker image.

Reply:
> BTW, since we have decided to go with community OpenCV version in the 2022.1.1 release, do we need OpenCV dependencies at all?

This PR is not related to OpenCV. For now, OpenCV is not added to the docker images and in not included in that script. OpenCV is added to the download script, in the documentation and in the demo tests automation.

Reply:
you are updating `pkgs_opencv_opt` variables, so why the change is not related to OpenCV? :) for what those dependencies otherwise?

Reply:
@ilya-lavrenov pkgs_opencv_opt is just a list of components related to demos using opencv. Here it is just the gstreamer. The change here is to drop the requirement of the license subscription and using gtreamer and its dependencies from centos8 repo.

Reply:
ok, this repo is for OpenVINO, it's not about OpenCV or demo? can we find a different place for these dependencies?